packages/appstore/src/components/loader/cashier-button-skeleton/cashierButtonSkeleton.scss

```
@@ -0,0 +1,11 @@
+@import '../skeletonLoader.scss';
+
+.cashier-button-skeleton-loader {
+    display: none;
+    height: 24px;
+    border-radius: 4px;
+    width: 59px;
+    @include mobile {
+        display: flex;
```

If you just want to make it appear only in mobile I suggest:
```suggestion
        display: block;
```